static/src/javascripts/projects/common/modules/experiments/tests/deeply-read-right-column.ts

```diff
@@ -0,0 +1,35 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const deeplyReadRightColumn: ABTest = {
+	id: 'DeeplyReadRightColumn',
+	author: '@dotcom-platform',
+	start: '2024-04-30',
+	expiry: '2024-07-28',
```

Comment:
It doesn't matter like it does for switches, but the 28th July is a Sunday. It almost definitely won't matter, but I'd recommend aligning the end date to the same as the switch (31st July).